apps/postybirb-ui/src/app/app-i18n-provider.tsx

```diff
@@ -22,29 +30,61 @@ export function AppI18nProvider(props: AppI18nProviderProps) {
       // eslint-disable-next-line no-param-reassign
       locale = locale ?? 'en';
       const { messages } = await import(`../lang/${locale}.po`);
+
+      const uppyLocale = uppyLocales[locale];
+      try {
+        i18n.uppy = (
+          await import(`../../public/uppy-i18n/${uppyLocale}.js`)
+        ).default;
```

Comment:
Instead of exporting separate variable/hook/context just set uppy right here, so all reactive components will receive update on the locale change